### PR TITLE
feat(common): added multiple animation support.

### PR DESCRIPTION
**Pull request by afsal-ali**
- [x]  common package multiple animation support

### CODE REVIEW

File: packages/common/src/dotlottie-player.ts [hunk 1]
```diff
@@ -294,15 +306,169 @@ export class DotLottiePlayer {
...
+    const anim = this._manifest?.animations.find((animation) => animation.id === activeId);
+
+    if (!anim) return;
+
+    this.render(anim);
+  }
+
+  public prev(options?: PlaybackOptions): void {
+    if (!this._manifest || !this._manifest.animations.length) {
+      throw new Error('[dotlottie]: manifest not found.');
```
Comment (samuelOsborne) [inline packages/common/src/dotlottie-player.ts:427]: maybe we can use util methods for easier Error management 

```

export const createError = (message: string): Error => {
  const error = new Error(`[dotLottie] ${message}`);

  return error;
};

export const logError = (message: string): void => {
  const error = `[dotLottie] ${message}`;

  console.error(error);
};

export const logWarning = (message: string): void => {
  const warning = `[dotLottie] ${message}`;

  console.warn(warning);
};

```

```
throw createError('...');
```

File: packages/react-player/src/hooks/use-dotlottie-player.ts [hunk 2]
```diff
@@ -15,10 +15,18 @@ interface UseDotLottiePlayerReturn {
   seeker: number;
 }
 
+export interface DotLottieRefProps {
+  getManifest: () => Manifest | undefined;
+  next: (options?: PlaybackOptions) => void;
+  play: (indexOrId?: string | number, options?: PlaybackOptions) => void;
+  prev: (options?: PlaybackOptions) => void;
```
Comment (samuelOsborne) [inline packages/react-player/src/hooks/use-dotlottie-player.ts:22]: could we call this previous?

**Comment by afsal-ali** [comment]
> irection and play

Thanks @samuelOsborne for finding these issues. 

1. solved the direction / playmode issues in the playground
2. Playground `next()` button has some hardcoded options which is overriding the manifest options. That's why it was playing earlier. It's removed now.

File: packages/react-player/src/hooks/use-dotlottie-player.ts (hunk 2 above)
Comment (afsal-ali) [reply packages/react-player/src/hooks/use-dotlottie-player.ts]: ✅ Done

File: packages/common/src/dotlottie-player.ts (hunk 1 above)
Comment (afsal-ali) [reply packages/common/src/dotlottie-player.ts]: ✅ Done

File: packages/common/src/dotlottie-player.ts [hunk 3]
```diff
@@ -294,15 +306,169 @@ export class DotLottiePlayer {
...
+    return this._activeAnimationId;
+  }
+
+  public reset(): void {
+    const activeId = this._manifest?.activeAnimationId;
+
+    if (!activeId) {
+      const anim = this._manifest?.animations[0];
+
+      if (!anim || !anim.id) return;
```
[the remark this answers is not in the document]
Comment (afsal-ali) [reply packages/common/src/dotlottie-player.ts]: ✅ Done

**Comment by samuelOsborne** [comment]
@afsal-ali Not sure if its because its the playground overriding the manifest options, but it feels like they arent being read even for a single animation

**Comment by afsal-ali** [comment]
> imation

Hi @samuelOsborne  I found the issue. It was because react-player has some default prop values. Made some updates to solve this issue. it should pick the correct options now. 
However discovered 1  issue with regards to `dotlottie-js`, The `dotlottieJs.manifest.animations` options doesn't match the  `options` set in the `.lottie` file.

**Pull request merged**